onnxruntime/python/tools/pytorch_export_contrib_ops.py

```diff
@@ -5,17 +5,20 @@
 Support for registering ONNX Runtime's built-in contrib ops with
 PyTorch-ONNX exporter (torch.onnx.export).
 """
-
 import typing
 
 try:
-    from torch.onnx import register_custom_op_symbolic
+    # TODO(justinchuby): Create a function to alert users when torch is not installed
```

Comment:
something similar exists at https://github.com/microsoft/onnxruntime/blob/master/orttraining/orttraining/python/training/ortmodule/__init__.py#L53-L74 for ORTModule (python frontend for ORT)

Reply:
Nice! Do you think we should consolidate them?

Reply:
Would be nice to use a common code for both places. In fact, I was reading and using `import something` embedded on a `try-except` is not actually good, because the module state can change during import.

You can borrow the idea from [here](https://github.com/pytorch/pytorch/blob/master/torch/testing/_internal/common_utils.py#L746-L760) for a safer solution and use in both places: